core/docz-utils/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "plugins": ["lodash"],
+  "presets": [
+    "@babel/preset-typescript",
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "current"
+        }
+      }
+    ]
+  ]
+}
```

Comment:
why did you have to add it? it shouldn't be needed

Reply:
Without this Jest was crashing on the TS syntax: it was not configured yet just because this package didn't have any tests. I copied it from other packages in this repo which already had some tests.

Reply:
I thought that probably it would be the reason, but thanks for explaining